File: app/src/main/java/net/mbonnin/arcanetracker/adapter/DeckEntryItem.java
package net.mbonnin.arcanetracker.adapter;

import net.mbonnin.arcanetracker.parser.EntityList;
import net.mbonnin.hsmodel.Card;


public class DeckEntryItem {

    public Card card;
    public int count;
    public boolean gift;

    public EntityList entityList = new EntityList();
}

File: app/src/main/java/net/mbonnin/arcanetracker/adapter/DeckEntryItem.kt
package net.mbonnin.arcanetracker.adapter

import net.mbonnin.arcanetracker.parser.EntityList
import net.mbonnin.hsmodel.Card


class DeckEntryItem(var card: Card, var count: Int = 0, var gift: Boolean = false , var entityList: EntityList = EntityList())